app/views/declaration/transport_details.scala.html

```diff
@@ -0,0 +1,60 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import controllers.declaration.routes._
+@import forms.declaration.TransportDetails
+@import services.Country
+@import uk.gov.hmrc.play.views.html._
+@import utils.RadioOption
+@(form: Form[TransportDetails])(implicit request: Request[_], appConfig: AppConfig, messages: Messages, countries: List[Country])
+
+@main_template(
+    title = messages("supplementary.transportInfo.title"),
+    appConfig = appConfig
+) {
+
+    @components.back_link("/customs-declare-exports/declaration/border-transport")
+
+    @helpers.form(TransportDetailsController.submitForm(), 'autoComplete -> "off") {
+
+        @components.error_summary(form.errors)
+
+        @components.page_title(Some("supplementary.transportInfo.title"), None)
+      
+         @components.input_text_autocomplete(form)(
+            field = form("meansOfTransportCrossingTheBorderNationality"),
+            label = messages("supplementary.transportInfo.meansOfTransport.crossingTheBorder.nationality.header"),
```

Comment:
And for the future, my propose is to make this message keys shorter

Reply:
not my code we have to move these later